internal/app/app.go

```diff
@@ -157,7 +160,7 @@ func NewCalendar(ctx context.Context, logger log.Logger) calendar.Calendar {
 	case CalendarNone:
 		return nil
 	default:
-		logger.Error(
+		logger.Warn(
```

Comment:
I would actually keep this one as Error, since there it could mean a typo in the config (ex: gogle_calendar) that could go unnoticed if it does not throw an error message.